website/docs/faq.md

```diff
@@ -41,16 +58,67 @@ Go to the [Plans](https://app.configcat.com/organization/plans) page and use the
 You need to have a [Billing Manager](organization.md#billing-manager-role) role to access the Plans page.
 
 ## Security
-### I can't login to ConfigCat, because I was asked a 2FA (Two-factor authentication) code that I don't know.
+### Are you ISO certified? 
+We can proudly say that as of 2022 May 31, ConfigCat has achieved the ISO/IEC 27001:2013 certification for Information Security Management Systems (ISMS).
 
-*Solution 1:* There might be an authenticator app on your phone that you can use to login to ConfigCat.
+### I'm configuring my firewall, which addresses should I whitelist?
+If possible, you can allow the whole "configcat.com" domain. Alternatively, you can manually whitelist the following addresses:
+- Global CDN: https://cdn-global.configcat.com
+- EU CDN: https://cdn-eu.configcat.com
+- The Public Management API: https://api.configcat.com
+- The Dashboard URL: https://app.configcat.com
+- API that serves the Dashboard: https://dashboard-api.configcat.com
 
-*Solution 2:* Contact your `Organization Admin`, and ask them to disable 2FA for your account until you set it up again.
+### I can't log in to ConfigCat, because I forgot my 2FA (Two-factor authentication) code.
```

Comment:
What happened here? The original answers were right.

Reply:
I just moved that line lower down, git formatted the change weirdly, but it's still in the text in its original form